pom.xml

```diff
@@ -25,9 +25,9 @@
   </modules>
 
   <scm>
-    <connection>scm:git:git@github.com:HotelsDotCom/beekeeper.git</connection>
-	<developerConnection>scm:git:git@github.com:HotelsDotCom/beekeeper.git</developerConnection>
-	<url>https://github.com/HotelsDotCom/beekeeper</url>
+    <connection>scm:git:git@github.com:ExpediaGroup/beekeeper.git</connection>
+    <developerConnection>scm:git:git@github.com:ExpediaGroup/beekeeper.git</developerConnection>
+    <url>https://github.com/HotelsDotCom/beekeeper</url>
```

Comment:
still says hotels

Reply:
Ah, I missed one! Thanks :D (it's gone now)